scripts/format_notebooks.py

```diff
@@ -124,6 +124,60 @@ def preprocess_cell(self, cell, resources, cell_index):
         return cell, resources
 
 
+class CloudRunnerPreprocessor(preprocessors.Preprocessor):
+    path_resource_name = "cloud_runner_path"
+    metadata_tag = "CloudRunner"  # tag for added cells so that we can find them easily
+    git_branch = "master"
+
+    colab_import_code = """\
+# install StellarGraph if running on Google Colab
+import sys
+if 'google.colab' in sys.modules:
+  !pip install -q stellargraph[demos]"""
```

Comment:
Minor, but maybe this could use [the `%pip` magic](https://ipython.readthedocs.io/en/stable/interactive/magics.html#magic-pip)?

```suggestion
  %pip install -q stellargraph[demos]"""
```

Reply:
good idea as this will be certain to be ties to the python kernel used by jupyter/colab